src/Validator/Param/ParamAbstract.php

```diff
@@ -6,30 +6,24 @@
 
 abstract class ParamAbstract implements ParamInterface
 {
+    //TODO: Drasko: should be private!
+
+    protected $value;
+
     /**
+     * @param mixed[] $meta
+     */
+    public function __construct(protected $name, private readonly Request $request, /**
```

Comment:
This is not formatted properly.
The long comment 

`* Possible options:
     *     default
     *     required
     *     separator
     *     type
     *     valid`
Should go to Docblock of __construct like:

`/**
  * @param $meta Possible options: default, required, separator, type, valid
  * /
`